### ISSUE TITLE
When calculating DNA sequence embeddings in batches, should they be normalized based on length?

### ISSUE THREAD

**Issue by zxzhang8** [jerryji1993/DNABERT#130]
When calculating the embedding representation of DNA sequences in batches, is it necessary to normalize the results using `attention_mask` and length?
Such as:
```python
dnas = [
        'ACGTAGCATCGGATCTATCTATCGACACTTGGTTATCGATCTACGAGCATCTCGTTAGC',
        'ACT'
]
model_name = "../resources/DNABERT-2-117M"
tokenizer = AutoTokenizer.from_pretrained(model_name, trust_remote_code=True)
config = AutoConfig.from_pretrained(model_name, trust_remote_code=True)
model = AutoModel.from_pretrained(model_name, config=config, trust_remote_code=True)

inputs = tokenizer(dnas, return_tensors="pt", truncation=True, padding="max_length", max_length=180)
hidden_states = model(input_ids=inputs["input_ids"], attention_mask=inputs["attention_mask"])[0]
mask = inputs["attention_mask"].unsqueeze(-1).type_as(hidden_states)
masked = hidden_states * mask
lengths = mask.sum(dim=1).clamp_min(1.0)
embedding_mean = masked.sum(dim=1) / lengths
```
or just:
```python
dnas = [
        'ACGTAGCATCGGATCTATCTATCGACACTTGGTTATCGATCTACGAGCATCTCGTTAGC',
        'ACT'
]
model_name = "../resources/DNABERT-2-117M"
tokenizer = AutoTokenizer.from_pretrained(model_name, trust_remote_code=True)
config = AutoConfig.from_pretrained(model_name, trust_remote_code=True)
model = AutoModel.from_pretrained(model_name, config=config, trust_remote_code=True)
inputs = tokenizer(dnas, return_tensors="pt", truncation=True, padding="max_length", max_length=180)
hidden_states = model(input_ids=inputs["input_ids"])[0]
torch.mean(hidden_states, dim=1)
```
Also, what if cls (`model(input_ids=inputs["input_ids"])[1]`) is used?
